cmd/internal/cli/inspect.go

```diff
@@ -533,7 +533,7 @@ func (c *command) addEnvironmentCommand() {
 func (c *command) addDefinitionCommand() {
 	deffile, err := inspectDeffilePartition(c.img)
 	if err == errNoSIFMetadata || err == errNoSIF {
-		c.addSingleFileCommand("Singularity", "deffile")
+		c.addSingleFileCommand("Apptainer", "deffile")
```

Comment:
The default definition file name should stay as Singularity.def for compatibility.

Reply:
of course you are correct, and these can be resolved by later Issue/PR rather than blocking this one

This is not a case where Athena needs to leap from Jupiter's head fully formed

Reply:
true -- however, please make this a generic issue to maintain backward compatibility containers ... amending the commit destroys the comment reference above.   Also, for the generic issue, we need to update the E2E tests to detect and enforce a backward compatibility rule.   Such E2E update does not belong in this PR

Reply:
deferred to #40